src/store/reducers/GetUserSlice.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import type { IUser } from 'API/auth-api';
+import { RootState } from 'store/store';
+import { fetchUser } from './ActionCreatorGetUser';
+
+export interface IGetUserState {
+    isLoading: boolean;
+    data: IUser | Record<string, unknown>;
```

Comment:
```suggestion
    data: IUser;
```

Множество значений `Record<string, unknown>` включает в себя множество значений `IUser`, поэтому при объединении результатом будет `Record<string, unknown>, что убивает строгость типизации